packages/app/src/app/components/CreateSandbox/CreateSandbox.tsx

```diff
@@ -335,10 +334,7 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = ({
                     }}
                     stopId="cloud-templates"
                   >
-                    <Stack gap={2}>
-                      <span>Cloud templates</span>
-                      <Badge>Beta</Badge>
-                    </Stack>
```

Comment:
Badge doesn't play well with the line height here. If it's important, I'll add a ticket later to add it back properly, but we will soon remove the beta anyway, so..